climatological_normal.json

```diff
@@ -0,0 +1,68 @@
+            "schema": {
+                "type": "object",
+                "subtype": "raster-cube"
+            }
+        },
+        {
+            "name": "frequency",
+            "description": "The time intervals to aggregate the average value for. The following pre-defined frequencies are supported:\n\n* `daily`: Day of the year\n* `monthly`: Month of the year\n* `climatology_period`: The period specified in the `climatology_period`.\n* `seasons`: Refers to three month periods of the calendar seasons (December - February, March - May, June - August, September - November).\n* `tropical_seasons`: Refers to the six month periods of the tropical seasons (November - April, Mai - October).",
+            "schema": {
+                "type": "string",
+                "enum": [
+                    "daily",
+                    "monthly",
+                    "seasons",
+                    "tropical_seasons",
+                    "climatology_period"
+                ]
+            }
+        },
+        {
+            "name": "climatology_period",
+            "description": "Closed temporal interval. The first element of the array is the first year to be fully included in the temporal interval. The second element is the last year to be fully included in the temporal interval. The default period is from 1981 until 2010 (both inclusive).",
+            "schema": {
+                "type": "array",
+                "subtype": "temporal-interval",
+                "minItems": 2,
+                "maxItems": 2,
+                "items": {
+                    "type": "string",
+                    "subtype": "year",
+                    "minLength": 4,
+                    "maxLength": 4,
+                    "pattern": "^\\d{4}$"
+                }
+            },
+            "default": [
+                "1981",
+                "2010"
+            ],
+            "optional": true
+        }
+    ],
+    "returns": {
+        "description": "A data cube with potentially cardinality, but the same resolution and number of dimensions as the original data cube. The temporal dimension has the following dimension labels:\n\n* `daily`: `1` - `365`\n* `monthly`: `1` - `12`\n* `climatology_period`: `climatology_period`\n* `seasons`: `djf` (December - February), `mam` (March - May), `jja` (June - August), `son` (September - November)\n* `tropical_seasons`: `ndjfma` (November - April), `mjjaso` (May - October)",
```

Comment:
for daily/monthly: are the labels strings or itegers?

Reply:
We implemented it for integers, but in general I didn't specify that to support cases where back-ends may only support strings.

Reply:
With the recent commits, all labels are strings now.